service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -32,15 +52,23 @@ default void bindServices(ServiceBinding<?>... serviceBindings) {
             }
         }
 
+        ServiceBinding<?> primaryServiceBinding = serviceBindings[0];
         // Bind the service info for the first one passed in
-        binder.bind(ServiceInfo.class).toProvider(new ServiceInfoProvider(serviceBindings[0].serviceInterface()));
+        binder.bind(ServiceInfo.class).toProvider(
+                new ServiceInfoProvider(
+                        primaryServiceBinding.serviceInterface(),
+                        Arrays
+                                .stream(serviceBindings)
+                                .map(ServiceBinding::serviceInterface)
+                                .toArray(Class[]::new)
+                ));
```

Comment:
Right ... I guess my point is that I don't understand the justification for excluding `MetricsService` from `ServiceInfo` when it's currently _included_ everywhere that might be changed to _use_ `ServiceInfo`.